cmake/sysbuild/partition_manager.cmake

```diff
@@ -498,6 +498,15 @@ foreach(d APP ${PM_DOMAINS})
     set(otp_size 764)  # 191 * 4
   endif()
 
+  sysbuild_get(${image_name}_CONFIG_SOC_SERIES_NRF54LX IMAGE ${image_name} VAR CONFIG_SOC_SERIES_NRF54LX KCONFIG)
+  if (DEFINED ${image_name}_CONFIG_SOC_SERIES_NRF54LX)
```

Comment:
I would omit `DEFINED` here (for consistency with existing code).

Reply:
applied